src/_imaging.c

```diff
@@ -518,6 +518,9 @@ getink(PyObject* color, Imaging im, char* ink)
        be cast to either UINT8 or INT32 */
 
     int rIsInt = 0;
+    if (PyTuple_Check(color) && PyTuple_Size(color) == 1) {
+        color = PyTuple_GetItem(color, 0);
+    }
```

Comment:
I think this currently makes `L` mode accept `tuple(tuple(10))` and `RGB` mode accept `tuple(tuple(255,255,255))`, which is probably not the intended behaviour.
```suggestion
    /* IMAGING_TYPE_UINT8 unpacks tuples later */
    if (im->type != IMAGING_TYPE_UINT8 && PyTuple_Check(color) && PyTuple_Size(color) == 1) {
        color = PyTuple_GetItem(color, 0);
    }
```